.github/actions/restore-node/action.yml

```diff
@@ -84,6 +88,11 @@ runs:
           sudo apt-get install libbsd-dev
         fi
         yarn install
+        # Replace the Endo packages with the ones built from the checked-out branch.
+        if test -e replacement-endo; then
+          mv replacement-endo ../
+          scripts/replace-packages.sh ../replacement-endo
+        fi
```

Comment:
Yeah I'd much prefer checking out `agoric-sdk` in a sub path in the first place.

Reply:
I made this renaming more obvious (in the action immediately following the checkout), so hope that helps now.